test_scripts/Smoke/API/020_SetDisplayLayout_PositiveCase_SUCCESS.lua

```diff
@@ -0,0 +1,230 @@
+
+local function getDisplayCapImageFieldsValues()
+	local names = {
+		"softButtonImage",
+		"choiceImage",
+		"choiceSecondaryImage",
+		"vrHelpItem",
+		"turnIcon",
+		"menuIcon",
+		"cmdIcon",
+		"graphic",
+		"showConstantTBTIcon",
+		"showConstantTBTNextTurnIcon"
+	}
+	local values = { }
+	for _, v in pairs(names) do
+		local item = {
+			imageResolution = {
+				resolutionHeight = 64,
+				resolutionWidth = 64
+			},
+			imageTypeSupported = {
+				"GRAPHIC_BMP",
+				"GRAPHIC_JPEG",
+				"GRAPHIC_PNG"
+			},
+			name = v
+		}
+		table.insert(values, item)
+	end
+  return values
+end
+
+local function getDisplayCapTextFieldsValues()
+  local names = {
+		"mainField1",
+		"mainField2",
+		"mainField3",
+		"mainField4",
+		"statusBar",
+		"mediaClock",
+		"mediaTrack",
+		"alertText1",
+		"alertText2",
+		"alertText3",
+		"scrollableMessageBody",
+		"initialInteractionText",
+		"navigationText1",
+		"navigationText2",
+		"ETA",
+		"totalDistance",
+		"navigationText",
+		"audioPassThruDisplayText1",
+		"audioPassThruDisplayText2",
+		"sliderHeader",
+		"sliderFooter",
+		"notificationText",
+		"menuName",
+		"secondaryText",
+		"tertiaryText",
+		"timeToDestination",
+		"turnText",
+		"menuTitle",
+		"locationName",
+		"locationDescription",
+		"addressLines",
+		"phoneNumber"
+	}
+	local values = { }
+	for _, v in pairs(names) do
+		local item = {
+			characterSet = "TYPE2SET",
+			name = v,
+			rows = 1,
+			width = 500
+		}
+		table.insert(values, item)
+	end
+  return values
+end
+
+local function getDisplayCapValues()
+	return {
+		displayType = "GEN2_8_DMA",
+		graphicSupported = true,
+		imageCapabilities =	{
+			"DYNAMIC",
+			"STATIC"
+		},
+		imageFields = getDisplayCapImageFieldsValues(),
+		mediaClockFormats =	{
+			"CLOCK1",
+			"CLOCK2",
+			"CLOCK3",
+			"CLOCKTEXT1",
+			"CLOCKTEXT2",
+			"CLOCKTEXT3",
+			"CLOCKTEXT4"
+		},
+		numCustomPresetsAvailable = 10,
+		screenParams = {
+			resolution = {
+				resolutionHeight = 480,
+				resolutionWidth = 800
+			},
+			touchEventAvailable =	{
+				doublePressAvailable = false,
+				multiTouchAvailable = true,
+				pressAvailable = true
+			}
+		},
+		templatesAvailable = {
+			"ONSCREEN_PRESETS"
+		},
+		textFields = getDisplayCapTextFieldsValues()
+	}
+end
+
+local function getRequestParams()
+	return { displayLayout = "ONSCREEN_PRESETS" }
+end
+
+local function getResponseParams()
+	return {
+		displayCapabilities = getDisplayCapValues(),
+		buttonCapabilities = getButCapValues(),
+		softButtonCapabilities = getSoftButCapValues(),
+		presetBankCapabilities = getPresetBankCapValues()
+	}
+end
+
+local function setDisplaySuccess(self)
+	local cid = self.mobileSession1:SendRPC("SetDisplayLayout", getRequestParams())
+	EXPECT_HMICALL("UI.SetDisplayLayout", getRequestParams())
+	:Do(function(_, data)
+			self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", getResponseParams())
+		end)
+	self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
```

Comment:
Sent parameters in HMI response must be checked also on mobile side. 
In mobile response can be added not all parameters from getResponseParams, because earlier was SDL issue with wrong processing of some parameters, need to check, and this parameters can be omitted for now.

Reply:
@GetmanetsIrina @aderiabin Done in 4f5ab90
Some options are commented for now since SDL has an issue and new defect has to be raised